content/applications/inventory_and_mrp/inventory/shipping_receiving/daily_operations/receipts_three_steps.rst

```diff
@@ -55,77 +55,61 @@ A :guilabel:`Receipt` smart button will appear in the top right, and the receipt
 with the purchase order. Clicking on the :guilabel:`Receipt` smart button will show the receipt
 order.
 
-.. image:: receipts_three_steps/three-step-purchase-receipt.png
+.. image:: receipts_three_steps/receipts-three-steps-smart-button.png
    :align: center
    :alt: After confirming a purchase order, a Receipt smart button will appear.
 
 Process a receipt
 -----------------
 
-One receipt and two internal transfers (one transfer to quality, and a subsequent transfer to stock)
-will be created once the purchase order is confirmed. To view these transfers, go to
-:menuselection:`Inventory --> Operations --> Transfers`.
+Once the purchase order (PO) is confirmed, a receipt (`WH/IN`) operation is generated and ready to
+process.
 
-.. image:: receipts_three_steps/three-step-transfers.png
-   :align: center
-   :alt: The status of the three receipt transfers will show which operation is ready and which ones
-         are waiting another operation.
-
-The status of the receipt transferring the product to the input location will be :guilabel:`Ready`,
-since the receipt must be processed before any other operation can occur. The status of the two
-internal transfers will be :guilabel:`Waiting Another Operation`, since the transfers cannot be
-processed until the linked step before each transfer is completed.
-
-The status of the first internal transfer to *quality* will only change to :guilabel:`Ready` when
-the receipt has been marked :guilabel:`Done`. The status for the second internal transfer to *stock*
-will be marked :guilabel:`Ready` only after the transfer to quality has been marked
-:guilabel:`Done`.
```

Comment:
Sanity check: These states, **Ready** and **Waiting Another Operation** are no longer relevant to 3-step receipts, correct?

Reply:
Yes, that's correct. 3-step receipts and deliveries now generate the operation only when it's ready, instead of listing all 3 at once like before.